lua/extn/sky/prelude.lua

```diff
@@ -0,0 +1,48 @@
+
+-- setup the global sky module for devices to use without having to redundently
+-- require/include the core
+
+local function import(target, module)
+  for k, v in pairs(module) do
+    target[k] = v
+  end
+end
+
+sky = {
+  __loaded = {},
+  __search = {norns.state.path, _path.code},
```

Comment:
remember we recently implemented the `paths` table for readability :)

Reply:
good point. i'll switch that.

Reply:
okay, everything uses `paths` now. thanks for catching that.